### ISSUE TITLE
xdebug provisioning error on ubuntu12 & php 5.3

### ISSUE THREAD

**Issue by danbohea** [hashbangcode/vlad#168]
If your settings contain the following preferences:

```
php_version: "5.3"
vlad_os: "ubuntu12"
```

Vlad will error when provisioning xdebug:

```
TASK: [php | xdebug | update xdebug configuration file] ***********************
failed: [nosferatu] => {"failed": true}
msg: Destination directory /etc/php5/mods-available does not exist
```

**Comment by mbarcia** [comment]
I've fixed this adding this

`- name: xdebug | update xdebug configuration file`
`  template: src=php_apache2_xdebug.ini.j2 dest=/etc/php5/apache2/conf.d/xdebug.ini`
`  when: xdebug_installed.changed and php_version == '5.3'`
`  tags: xdebug`
`  sudo: true`

**Comment by mbarcia** [comment]
Dan,

This patch should fix the issue (only changes the php version test). HTH, cheers.

diff --git a/vlad/playbooks/roles/php/tasks/debian_xdebug.yml b/vlad/playbooks/roles/php/tasks/debian_xdebug.yml
index 7fd21bc..8b96ced 100644
--- a/vlad/playbooks/roles/php/tasks/debian_xdebug.yml
+++ b/vlad/playbooks/roles/php/tasks/debian_xdebug.yml
@@ -27,7 +27,7 @@
 
 - name: xdebug | update xdebug configuration file
   template: src=php_apache2_xdebug.ini.j2 dest=/etc/php5/mods-available/xdebug.ini
-  when: xdebug_installed.changed and php_version != '5.4'
+  when: xdebug_installed.changed and php_version != '5.3'
   tags: xdebug
   sudo: true

**Comment by philipnorton42** [comment]
Thanks Mariano. I've applied that fix to dev :)

**Comment by philipnorton42** [comment]
This has now been fixed and released in v1.0.4.
